package/MDAnalysis/analysis/distances.py

```diff
@@ -42,7 +42,7 @@
            'contact_matrix', 'dist', 'between']
 
 import numpy as np
-import scipy.sparse
+from scipy import sparse
```

Comment:
That's ok, I can live with it.

If I take only few things I like fully qualified imports but this one is fine if you like it better.